cmake/secp256k1.cmake

```diff
@@ -48,4 +48,5 @@ function(add_secp256k1 subdir)
   set_target_properties(secp256k1 PROPERTIES
     EXCLUDE_FROM_ALL TRUE
   )
+  set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Above, line 9, it's being set to OFF.

Reply:
Setting it in the function scope should have no effect after the function call.